vmdk_plugin/utils/plugin_server/plugin_server_linux.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package plugin_server
```

Comment:
Why would the linux, windows, shared plugin servers get built into each plugin? Pls. separate into separate packages for vmdk, windows, shared. And instantiate the appropriate one from main().